consensus/types/src/light_client_update.rs

```diff
@@ -24,20 +24,36 @@ pub const CURRENT_SYNC_COMMITTEE_INDEX: usize = 54;
 pub const NEXT_SYNC_COMMITTEE_INDEX: usize = 55;
 pub const EXECUTION_PAYLOAD_INDEX: usize = 25;
 
+pub const FINALIZED_ROOT_INDEX_ELECTRA: usize = 169;
+pub const CURRENT_SYNC_COMMITTEE_INDEX_ELECTRA: usize = 86;
+pub const NEXT_SYNC_COMMITTEE_INDEX_ELECTRA: usize = 87;
+
 pub type FinalizedRootProofLen = U6;
 pub type CurrentSyncCommitteeProofLen = U5;
 pub type ExecutionPayloadProofLen = U4;
-
 pub type NextSyncCommitteeProofLen = U5;
 
+pub type FinalizedRootProofLenElectra = U7;
+pub type CurrentSyncCommitteeProofLenElectra = U6;
+pub type NextSyncCommitteeProofLenElectra = U6;
+
 pub const FINALIZED_ROOT_PROOF_LEN: usize = 6;
 pub const CURRENT_SYNC_COMMITTEE_PROOF_LEN: usize = 5;
 pub const NEXT_SYNC_COMMITTEE_PROOF_LEN: usize = 5;
 pub const EXECUTION_PAYLOAD_PROOF_LEN: usize = 4;
 
+pub const FINALIZED_ROOT_PROOF_LEN_ELECTRA: usize = 7;
+pub const NEXT_SYNC_COMMITTEE_PROOF_LEN_ELECTRA: usize = 6;
+pub const CURRENT_SYNC_COMMITTEE_PROOF_LEN_ELECTRA: usize = 6;
+
+pub type Proof = Vec<Hash256>;
```

Comment:
I think maybe we should use a more specific name for this type, as there are many kinds of proofs floating around, especially with peer DAS and EIP4844.

```suggestion
pub type MerkleProof = Vec<Hash256>;
```